roles/commcare_analytics/tasks/rabbitmq.yml

```
@@ -28,13 +28,11 @@
 
 - name: Place config file
   template:
-    src: "{{ item.src }}"
-    dest: "/etc/rabbitmq/{{ item.dest }}"
+    src: rabbitmq/rabbitmq.conf.j2
```

I believe that this and the following line should have quotation marks included.

```suggestion
    src: "rabbitmq/rabbitmq.conf.j2"
```

---

Hey @zandre-eng 
By default any value is considered a string so the quotes are redundant. Though some consistency should be good.

---

Ah, it's fine to leave as is then.